src/main/scala/de/m7w3/signal/AccountHelper.scala

```diff
@@ -13,11 +13,10 @@ import org.whispersystems.libsignal.util.{KeyHelper, Medium}
 import org.whispersystems.signalservice.api.SignalServiceAccountManager
 import org.whispersystems.signalservice.internal.util.Base64
 
-case class AccountHelper(userId: String){
+case class AccountHelper(userId: String, password: String = Util.getSecret(20)){
```

Comment:
It is static. At the next run of the application (after registration) the password from the DB (the same as the first time) is passed here as an argument instead of the default value. 

I can call `Util.getSecret(20)` outside and pass the password explicitly the first time